contrib/win32/install/build/build-msi.ps1

```diff
@@ -0,0 +1,224 @@
+[CmdletBinding()]
+param(
+	[Parameter(Mandatory)]
+	[ValidateScript( { Test-Path -LiteralPath $_ -PathType Leaf })]
+	[string]
+	$ProjectPath,
+
+	[Parameter(Mandatory)]
+	[ValidateScript( { Test-Path -LiteralPath $_ -PathType Container })]
+	[string]
+	$SourceDirectory,
```

Comment:
We already have a build pipeline which does compilation, sign all artifacts and finally creates a zip payload.
If I want to add one more task to create the MSI package then I just need to call this script with $projectpath as openssh-install.wixproj, $sourcedirectory as <artifact_bin_folder>, $destination as <dest_folder_for_final_msi>, $version is derived from the executables and platform/configuration?

Do I need to change any GUIDs? I saw there is some discussion about hardcoding GUID vs generating GUID automatically.